sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeCustomFormsAsync.java

```diff
@@ -43,21 +43,23 @@ public static void main(String[] args) throws IOException {
         File sourceFile = new File("../formrecognizer/azure-ai-formrecognizer/src/samples/java/sample-forms/"
             + "forms/Invoice_6.pdf");
         byte[] fileContent = Files.readAllBytes(sourceFile.toPath());
-        InputStream targetStream = new ByteArrayInputStream(fileContent);
-        String modelId = "{modelId}";
+        PollerFlux<OperationResult, List<RecognizedForm>> recognizeFormPoller;
+        try (InputStream targetStream = new ByteArrayInputStream(fileContent)) {
```

Comment:
what is the purpose of using try() but without catch()?

Reply:
This is an auto closeable try block. This will make sure the stream is closed out of the block.